.github/workflows/test.yml

```diff
@@ -1,10 +1,17 @@
 name: Run test suites
+run-name: >-
+  ${{ (github.event.inputs.runId && format('Run test suites - run id {0}', github.event.inputs.runId))
+      || (github.event.inputs.linuxBuildFile && format('Run test suites - linux build {0}', github.event.inputs.linuxBuildFile))
+      || '' }}
```

Comment:
according to the GH docs, setting the run-name to empty string will cause it to take on the default name for that sort of event.  So for PRs, it'll keep the existing behavior.